### ISSUE TITLE
Request for Access to Training Data Used for Fine-Tuning Mask R-CNN Model

### ISSUE THREAD

**Issue by openingelevator** [alfworld/alfworld#109]
I recently came across your paper where you mentioned fine-tuning a Mask R-CNN model, pre-trained on COCO, using a dataset of 50k images. I was able to successfully download and utilize the trained model provided in your GitHub repository, which has been incredibly helpful for my work. However, I noticed that neither the repository nor the paper includes information on how to access the annotated training data used for fine-tuning.

<img width="720" alt="Image" src="https://github.com/user-attachments/assets/6a5e39b1-00b0-4f6a-8945-1bac08cdf2e5" />

I would greatly appreciate it if you could share these annotated datasets, as I am interested in using them to fine-tune other models for my research. If sharing the data is not feasible due to any constraints, I would still like to express my sincere gratitude for your excellent work and the resources you have already made available.

**Comment by MarcCote** [comment]
Hi! Thanks for reaching out. I was not involved in that part of the project (looping in @MohitShridhar just in case). As a starting point, I would look at this script https://github.com/alfworld/alfworld/blob/master/scripts/train_mrcnn.py

**Comment by openingelevator** [comment]
Thank you for your response and help! This is really helpful.
